README.md

```diff
@@ -51,6 +51,19 @@ models:
     +schema: my_new_schema_name # leave blank for just the target_schema
 ```
 
+This package allows users to include addtiional columns to the task table.  You must include any additional columns in both the `asana` and `asana_source` pass-through variables to ensure the downstream columns are present.
```

Comment:
```suggestion
This package allows users to include additional columns to the source task table.  To do this, include any additional columns to the `asana_source` pass-through variables to ensure the downstream columns are present.
```

Reply:
Looks like we don't need any new variables in the transformation package, as the `asana__task` model selects all columns from the staging source model.